test/bdd/README.md

```diff
@@ -0,0 +1,4 @@
+# Steps to run this BDD
```

Comment:
Don't use the directory `bdd`.
Rename: `test` -> `tests`
- All BDD will go inside the tests directory
-  Later if tests directory have to many files you can categorize all files in a separate directory inside `tests` directory itself